app/classifier.py

```diff
@@ -364,7 +363,10 @@ def classify() -> Any:
 
     c = ClassifierCache.get(app.config['BASE_CLASSIFIER_DIR'], args['model'])
     with c.lock:
-        results = c.classify(data['seqs'])
+        if 'probs' in args or ('probs' in data and data['probs']):
```

Comment:
can you add an explanatory comment here about what the condition really signifies?